pom.xml

```diff
@@ -704,19 +710,40 @@ limitations under the License.
 									<configuration>
 										<artifactItems>
 											<artifactItem>
-												<groupId>org.springframework.cloud</groupId>
-												<artifactId>spring-cloud-build-docs</artifactId>
-												<version>${spring-cloud-build.version}</version>
-												<classifier>sources</classifier>
-												<type>jar</type>
-												<overWrite>false</overWrite>
-												<outputDirectory>${docs.resources.dir}</outputDirectory>
+												<groupId>io.spring.docresources</groupId>
```

Comment:
Here we setup what we would like to do, once someone wants to use the download dependency plugin